internal/server/logger.go

```diff
@@ -0,0 +1,123 @@
+package server
```

Comment:
Maybe make this a new package, with some high-level documentation what it does and what it expects from its environment, ideally with references to documentation? Is this specific to CloudWatch? cloud.redhat.com? OpenShift?

Reply:
Do you want this documentation inline or rather in the README, not sure what conventions are in osbuild

Reply:
Both? ;) I mean, technical details in the package and enough info to set things up in the README? Ideally the code should be explained in isolation though (in case it is used in other places or implements features we don't yet/stop using).